### PR TITLE
feat: add an event_queue cleaning job

### PR DESCRIPTION
**Pull request by adityathebe**
Resolves: https://github.com/flanksource/incident-commander/issues/411

### CODE REVIEW

File: jobs/event_queue.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import (
+	"time"
+
+	"github.com/flanksource/commons/logger"
+	"github.com/flanksource/duty/models"
+	"github.com/flanksource/incident-commander/api"
+	"github.com/flanksource/incident-commander/db"
+)
+
+// CleanupEventQueue deletes state records in the `event_queue` table
```
Comment (yashmehrotra) [inline jobs/event_queue.go:12]: stale*

File: jobs/event_queue.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+func CleanupEventQueue() {
```
Comment (yashmehrotra) [inline jobs/event_queue.go:13]: Rename to CleanupPushEvents or something

**Review by yashmehrotra — APPROVED** [review]
LGTM

File: jobs/event_queue.go (hunk 2 above)
Comment (adityathebe) [reply jobs/event_queue.go:13]: I think we would want to clean up other event types in the queue as well.

Comment (adityathebe) [reply jobs/event_queue.go:13]: Added deletion of other events as well

File: jobs/event_queue.go [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+		"check_statuses":                 time.Hour * 24 * 7,
+	}
+
+	for table, age := range pushQueueSchedule {
+		result := db.Gorm.Debug().Exec("DELETE FROM event_queue WHERE name = 'push_queue.create' AND properties->>'table' = ? AND NOW() - created_at > ?", table, age)
+		if result.Error != nil {
+			logger.Errorf("Error cleaning up push_queue events for table=%s: %v", table, result.Error)
+			jobHistory.AddError(result.Error.Error())
+		} else {
+			logger.Debugf("Cleaned up %d push_queue events for table=%s", result.RowsAffected, table)
```
Comment (moshloop) [inline jobs/event_queue.go:40]: This should probably be a Warn if row coount > 0

**Pull request merged**